frappe/custom/doctype/customize_form/customize_form.json

```
@@ -25,6 +25,10 @@
   "show_title_field_in_link",
   "image_field",
   "search_fields",
+  "auto_name_section_section",
+  "autoname",
+  "cb_autoname_column",
+  "file_auto_name",
```

```suggestion
  "file_auto_name",
```

Can we rename this to file_autoname for the sake of consistency